metadata_backend/conf/conf.py

```diff
@@ -89,3 +89,21 @@ def create_db_client() -> AsyncIOMotorClient:
 
 # 4) Set frontend folder to be inside metadata_backend modules root
 frontend_static_files = Path(__file__).parent.parent / "frontend"
+
+
+# 5) Set up configurations for AAI server
+
+aai_config = {
+    "client_id": os.getenv("CSC_AAI_CLIENT_ID", "public"),
+    "client_secret": os.getenv("CSC_AAI_CLIENT_SECRET", "secret"),
+    "domain": os.getenv("BASE_URL", "http://localhost:5430"),
+    "scope": "openid profile email",
+    "iss": os.getenv("AUTH_URL", ""),
+    "aud": "aud1",
+    "callback_url": f'{os.getenv("BASE_URL", "http://localhost:5430")}/callback',
+    "auth_url": f'{os.getenv("OIDC_URL", "")}/authorize',
```

Comment:
When I was testing this branch with the `feature/login-link` branch of the frontend repo, I ran into an error: `Oops! Page /authorize could not be found.`

I assume it's caused by this and that this should instead be:
```suggestion
    "auth_url": f'{os.getenv("OIDC_URL", "")}/aai',
```

Reply:
nope, that is correct. that is oidc endpoint configuration 
I suspect you were missing something from configuration in docker-compose or did not run the frontend at same url as backend. You need to build and start the full image not the dev one.
In the `Dockerfile` you need to `RUN git checkout feature/login-link` after line 9 to build the image with that branch